src/components/ClubHome/Manager/Members.js

```diff
@@ -1,27 +1,41 @@
-import React from "react";
+import React, { useRef } from "react";
 import styles from "../../../styles/Club/Home/Manager/Manager.module.scss";
 import MembersList from "./MembersList";
 import MembersHeader from "./MembersHeader";
 import MembersPreface from "./MembersPreface";
 
-export const Members = ({ members, leader }) => {
+export const Members = ({
+  members,
+  leader,
+  onLeaderChange,
+  onApplyAuthClick,
+  onBoardAuth,
+  refArr,
+  changeMembersAuth,
+}) => {
   return (
     <div>
       <MembersHeader members={members.length} />
       <MembersPreface />
-      {members.map((el, i) => {
+      {members.map((el, index) => {
```

Comment:
```suggestion
      {members.map((member, index) => {
```
이런식으로 복수형, 단수형 변수명이 좋다고 들어서 리뷰 남깁니다